### PR TITLE
Add columns and obs(var)_keys

### PR DESCRIPTION
**Pull request by rm1113**
PR addresses issues #1 #2 . 

Also PR provides more intuitive raw layer handle, now to read raw.var the method `cap_adata.raw.read_var()` must be called. The new class BaseLayerMatrixAndDf was implemented to collect the shared functionality of main and raw layers.

If user want's to read more columns in addition to columns which are in memory already he can run `read_obs(columns)`. If user want to overwrite in-memory DataFrame with columns from AnnData he can  run `read_obs(columns, flush=True)`

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (siberianisaev) [inline .idea/misc.xml]: @rm1113 please update .gitignore file

File: README.md [hunk 2]
```diff
@@ -5,31 +5,65 @@ CAP-AnnData enriches the AnnData ecosystem by offering tailored functionalities
...
+pip install pytest
+pytest test
+```
 
 Make sure Python 3.9 or newer is used, along with all requirements specified in requirements.txt
 
 ## How-TO:
 
-#### 1. Read AnnData File Dataframes
+#### 1. Access AnnData File Dataframes
```
Comment (siberianisaev) [inline README.md]: just minor fix may be `DataFrames`

File: cap_anndata/cap_anndata.py [hunk 3]
```diff
@@ -91,31 +75,110 @@ def _read_df(self, h5_group: h5py.Group, columns: List[str]) -> CapAnnDataDF:
...
+
+class RawLayer(BaseLayerMatrixAndDf):
+    def __init__(self, h5_file: h5py.File):
+        super().__init__(h5_file, path_to_content="/raw/")
+        self._var: CapAnnDataDF = None
+
+    @property
+    def var(self) -> CapAnnDataDF:
+        if self._var is None:
+            self._var = self._lazy_df_load("var")
```
Comment (siberianisaev) [inline cap_anndata/cap_anndata.py:106]: We will need to move all strings like `var`, `obs`, `uns` to constants/enum, feel free to add refactoring issue

File: cap_anndata/cap_anndata.py [hunk 4]
```diff
@@ -91,31 +75,110 @@ def _read_df(self, h5_group: h5py.Group, columns: List[str]) -> CapAnnDataDF:
...
+        return self._var
+
+    def read_var(self, columns: List[str] = None, flush: bool = False) -> None:
```
Comment (siberianisaev) [inline cap_anndata/cap_anndata.py]: may be better `rewrite`/`refresh` than `flush`?

File: cap_anndata/cap_anndata.py [hunk 5]
```diff
@@ -124,19 +187,21 @@ def overwrite(self, fields: List[str] = None) -> None:
...
                 if entity is None:
                     continue
 
-                key = key.replace(".", '/') if key == "raw.var" else key
+                key = key.replace(".", "/") if key == "raw.var" else key
 
                 for col in entity.columns:
                     self._write_elem_lzf(f"{key}/{col}", entity[col].values)
-                self._file[key].attrs['column-order'] = entity.column_order
+                self._file[key].attrs["column-order"] = entity.column_order
```
Comment (siberianisaev) [inline cap_anndata/cap_anndata.py]: `column-order` could be moved to constants also

**Review by siberianisaev — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (rm1113) [reply README.md]: thanks

File: cap_anndata/cap_anndata.py (hunk 4 above)
Comment (rm1113) [reply cap_anndata/cap_anndata.py]: renamed to reset

**Pull request merged**